spikingjelly/activation_based/op_counter/neuromc/core.py

```
@@ -142,725 +194,1247 @@ def _resolve_loss_fn(loss_fn: Callable | None):
+        try:
+            for handle in self._hook_handles:
+                handle.remove()
+            self._hook_handles.clear()
+            if self._bound_model is not None:
+                for module in self._bound_model.modules():
+                    if hasattr(module, "_neuromc_last_input"):
+                        delattr(module, "_neuromc_last_input")
+            self._model_bound = False
+            self._bound_model = None
+        finally:
+            return self._trace_mode.__exit__(exc_type, exc, tb)
```

Ruff (0.15.12)

[warning] `return` inside `finally` blocks cause exceptions to be silenced

(B012)